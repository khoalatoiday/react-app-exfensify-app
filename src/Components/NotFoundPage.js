import { Link } from "react-router-dom";
import React from "react";
const NotFoundPage = () => (
    <p>
      404 Page <Link to="/">Go Home</Link>
    </p>
  );

export {NotFoundPage as default}