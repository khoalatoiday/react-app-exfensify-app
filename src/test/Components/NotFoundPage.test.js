import React from "react";
import { shallow } from "enzyme";
import NotFoundPage from "../../Components/NotFoundPage";

test("Should render not found page", ()=>{
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot()
})