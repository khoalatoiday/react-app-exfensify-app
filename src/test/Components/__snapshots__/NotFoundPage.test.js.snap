// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render not found page 1`] = `
<p>
  404 Page 
  <Link
    to="/"
  >
    Go Home
  </Link>
</p>
`;
